package.json

```diff
@@ -1,31 +1,26 @@
 {
-    "name": "gtile-extension",
-    "version": "0.0.1",
-    "description": "gTile Gnome extension - a tiling extension",
-    "license": "GPL v2+",
-    "devDependencies": {
-        "@types/chai": "^4.3.4",
-        "@types/jasmine": "^4.3.1",
-        "@types/jsdom": "^20.0.1",
-        "@types/mocha": "^10.0.1",
-        "@types/yargs": "^17.0.19",
-        "chai": "^4.3.7",
-        "jasmine": "^4.5.0",
-        "jasmine-core": "^4.5.0",
-        "jsdom": "^21.0.0",
-        "karma": "^6.4.1",
-        "karma-chrome-launcher": "^3.1.1",
-        "karma-firefox-launcher": "^2.1.2",
-        "karma-jasmine": "^5.1.0",
-        "karma-requirejs": "^1.1.0",
-        "karma-sourcemap-loader": "^0.3.8",
-        "mocha": "^10.2.0",
-        "requirejs": "^2.3.6",
-        "rollup": "^3.10.0",
-        "typescript": "^4.9.4",
-        "yargs": "^17.6.2"
-    },
-    "scripts": {
-    },
-    "type": "commonjs"
+  "name": "gnome-shell-extension-gtile",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "gTile Gnome extension",
+  "module": "dist/extension.js",
+  "author": "Markus Schanz <markus@schanz.id>",
+  "license": "GPL-2.0-or-later",
+  "devDependencies": {
+    "@schnz/adw-1": "^1.4.0-3.2.2",
+    "@schnz/gnome-shell": "^3.0.0",
```

Comment:
I'm not able to find these with some basic googling, and my normal flow for looking them up on NPM isn't working because they're not published to NPM.

I get a little nervous about supply chain attacks coming from unknown dependencies and their transitive depependencies.

Reply:
See comment above.